### PR TITLE
Log server version

### PR DESCRIPTION
**Pull request by dev-gto**
When running different krill instances on several machines using either docker or rsyslog, it is useful to log each version at startup, as well as their configuration filenames

### CODE REVIEW

File: src/daemon/config.rs [hunk 1]
```diff
@@ -215,6 +217,12 @@ impl Config {
         let config_file = matches
             .value_of("config")
             .unwrap_or("./defaults/krill.conf");
+        format!("{}", config_file)
+    }
+
+    /// Creates the config (at startup). Panics in case of issues.
+    pub fn create() -> Result<Self, ConfigError> {
+        let config_file = &*Self::get_config_filename();
```
Comment (timbru) [inline src/daemon/config.rs]: Why do we need &* here?

**Comment by timbru** [comment]
Thank you for the pull request.

Could you move the constants to src/lib.rs and have only one VERSION constant? And change the name for the krill app. I.e. add the following:

```
pub const KRILL_VERSION: &str = "0.2.1";
pub const KRILL_SERVER_APP: &str = "Krill";
pub const KRILL_CLIENT_APP: &str = "Krill Client";
```

Other than that I left a comment regarding the use of '&*'. Can't that just be left out?

I am happy to merge it when these things are changed.

File: src/daemon/config.rs (hunk 1 above)
Comment (dev-gto) [reply src/daemon/config.rs]: actually I'm a newbie in rust world... I will fix this

**Comment by dev-gto** [comment]
I've just finished the changes... Please let me know if there are any issues left.

File: src/lib.rs [hunk 2]
```diff
@@ -1,3 +1,7 @@
+pub const KRILL_VERSION: &str = "0.2.1";
+pub const KRILL_SERVER_APP: &str = "NLnet Labs RRDP Server";
```
Comment (timbru) [inline src/lib.rs]: Can you name the server "Krill"

File: src/daemon/krillserver.rs [hunk 3]
```diff
@@ -82,6 +84,8 @@ impl KrillServer {
         let token = &config.auth_token;
         let ca_refresh_rate = config.ca_refresh;
 
+        info!("{} v{}", KRILL_SERVER_APP, KRILL_VERSION);
```
Comment (timbru) [inline src/daemon/krillserver.rs]: Can you change these three lines to:
```
info!("Starting {} v{}", KRILL_SERVER_APP, KRILL_VERSION);
info!("{} uses configuration file: {}", KRILL_SERVER_APP, Config::get_config_filename());
info!("{} uses service uri: {}", KRILL_SERVER_APP, service_uri);
```

Comment (dev-gto) [reply src/daemon/krillserver.rs]: ok, done

**Comment by timbru** [comment]
Thanks! I will merge as soon as the Travis build comes through. (network in commuter train permitting).

**Pull request merged**